lib/views/register.dart

```diff
@@ -0,0 +1,213 @@
+import 'package:flutter/material.dart';
+import 'package:thirumathikart_app/widgets/button/register_button.dart';
+
+class RegisterPage extends StatelessWidget {
+  const RegisterPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double deviceHeight = MediaQuery.of(context).size.height;
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: ConstrainedBox(
+            constraints: BoxConstraints(
+              minHeight: deviceHeight,
+            ),
+            child: Container(
+              padding: const EdgeInsets.all(20),
+              decoration: const BoxDecoration(
+                color: Color.fromRGBO(251, 242, 207, 1),
+              ),
+              child: Center(
+                child: Card(
+                  elevation: 12,
+                  child: Padding(
+                    padding: const EdgeInsets.all(10),
+                    child: Column(
```

Comment:
TextFields can be iterated by have an array of json and passing the value to the common textfield widget